src/client/components/checkout/Footer.stories.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+import { MemoryRouter, Route, RouterProps } from 'react-router-dom'
+import { LocaleLabel } from 'l10n/locales'
+import {
+  mockedQuoteCartId,
+  mockedPriceQueryResponse,
+} from 'utils/testData/priceQueryMock'
+import { TextKeyProvider, TranslationsLocale } from 'utils/textKeys'
+import { PriceDocument, Locale } from 'data/graphql'
+import { localePathPattern } from 'l10n/localePathPattern'
+import { Footer, Props as FooterProps } from './Footer'
+
+type StoryProps = {
+  localePath: LocaleLabel
+} & FooterProps
+
+const storyMeta: Meta<StoryProps> = {
+  title: 'Components/Checkout/Footer/Footer',
+  component: Footer,
+  args: {
+    buttonText: 'Continue to payment',
+    buttonOnClick: () => {
+      console.log('👉 Click!')
+    },
+    localePath: 'no-en',
+  },
+  parameters: {
```

Comment:
To make the controls a bit easier to use you could predefine a dropdown with locales :) 
```suggestion
  argTypes: {
    localePath: {
      name: 'locale',
      options: ['se', 'se-en', 'no', 'no-en', 'dk', 'dk-en'],
      control: { type: 'select' },
    },
  },
  parameters: {
```

Reply:
OMG thank you, that's what I really wanted but didn't understand how to do that (and just gave up without googling lol) 👌 🙏